src/Order.js

```diff
@@ -0,0 +1,19 @@
+import React, { useState } from "react";
+import RestaurantButton from "./ResturantButton";
+
+function Order(props) {
+  const [orders, setOrders] = useState(0);
+  //const pizzas = 0;
+
+  function orderOne() {
+    setOrders(orders + 1);
+  }
+  return (
+    <li>
+      <RestaurantButton clickFunc={orderOne} />
+      {props.orderType}: {orders}
+    </li>
```

Comment:
Nice passing of the function here. Would be a good idea to change the name of 'clickFunc' to something which is more explanatory (for example orderOne = {orderOne} ) Someone else reading your code may wonder what clickFunc relates to when looking at the child component.